### PR TITLE
jdk8,11 Ignore "allow" and "disallow" set in java.security.manager

### PR DESCRIPTION
**Pull request by pshipton**
Issue https://github.com/eclipse-openj9/openj9/issues/18401

### CODE REVIEW

File: jcl/src/java.base/share/classes/java/lang/System.java [hunk 1]
```diff
@@ -1188,18 +1188,19 @@ static void checkTmpDir() {
 
 static void initSecurityManager(ClassLoader applicationClassLoader) {
 	String javaSecurityManager = internalGetProperties().getProperty("java.security.manager"); //$NON-NLS-1$
-	/*[IF JAVA_SPEC_VERSION > 11]*/
 	if ("allow".equals(javaSecurityManager)) {
```
Comment (keithc-ca) [inline jcl/src/java.base/share/classes/java/lang/System.java:1191]: This method should either be annotated with `@SuppressWarnings("nls")` or `//$NON-NLS-1$` should be used consistently.

**Comment by pshipton** [comment]
Updated it.

**Comment by keithc-ca** [comment]
Jenkins test sanity alinux jdk8,jdk11,jdk17,jdk21

**Comment by keithc-ca** [comment]
The only test failure was
```
org.eclipse.openj9.criu.JVMRestoreException: Blocking operation is not allowed in CRIU single thread mode.
```
which is unrelated to this change.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**